app/capabilities/capabilities.tsx

```diff
@@ -9,6 +9,8 @@ export class Capabilities {
   auth: string;
   anonymous: boolean;
   createOrg: boolean;
+  // TODO: Base this on the buildbuddyConfig
+  invocationSharing: boolean = true;
```

Comment:
does this need to be driven by localstorage or something?

Reply:
This is the flag that was going to be driven by the config file, which I hard-coded to true for now. There's already a localStorage check in the component itself.

But I think it makes more sense for this capability to contain the localStorage check, and then in the component we check whether invocation sharing is enabled by the group.

Updated the PR.